cmd/process-agent/common/tagger_params.go

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package common provides a set of common functions used on different commands
+package common
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	tagger "github.com/DataDog/datadog-agent/comp/core/tagger/def"
+	"github.com/DataDog/datadog-agent/comp/core/tagger/types"
+	"github.com/DataDog/datadog-agent/pkg/api/security"
+	"github.com/DataDog/datadog-agent/pkg/config/env"
+)
+
+// DualTaggerParams returns the params use inside the main agent
+func DualTaggerParams() (tagger.DualParams, tagger.Params, tagger.RemoteParams) {
```

Comment:
Could we merge this one with the one for cmd/agent (having functions with the same name for each process init phase is confusing I think).

Reply:
Each of them are for their particualar `cmd/*/` package. I don't think merging make much sense